chai/chai.d.ts

```diff
@@ -25,6 +28,28 @@ declare module Chai {
         (target: any, message?: string): Assertion;
     }
 
+    interface ShouldAssertion {
+        equal(val1: any, val2: any, msg?: string): void;
+        Throw: ShouldThrow;
+        throw: ShouldThrow;
+        exist(val: any, msg?: string): void;
+    }
+
+    interface Should extends ShouldAssertion {
+        not: ShouldAssertion;
+        fail(actual: any, expected: any, message?: string, operator?: string): void;
+    }
+
+    interface ShouldThrow {
+        (actual: Function): void;
+        (actual: Function, expected: string, message?: string): void;
```

Comment:
Happy with that - hadn't done it to stick with the current file conventions, but will make that change (and for `constructor: Error|Function` in the same set of overloads)